### DISCUSSION TITLE
Please consider using FerretDB instead of MongoDB as database

### DISCUSSION THREAD

**Discussion by the-digital-anarchist** [payloadcms/payload#2491] [Feature Requests & Ideas]
I'm going to be honest here, my knowledge of MongoDB and FerretDB are limited let alone how Payload is working together with it.

What I know is that MongoDB is not open-source anymore and FerretDB[1] just launched as a "truly Open Source MongoDB alternative" to be production ready. Would this be a good option to switch Payload to as default database?

[1] https://blog.ferretdb.io/ferretdb-1-0-ga-opensource-mongodb-alternative/

**Comment by jmikrut** [comment]
I have seen this recently too! I think we should be able to work with FerretDB already. You should try it. We'll get around to it eventually as well but I believe their mission is to be compatible with Mongo APIs so if they are doing their job right it could be plug-and-play, similar to DocumentDB and Azure Cosmos MongoDB.

**Reply by the-digital-anarchist** [reply]
Oke sounds exciting, I'll give it a shot if I have time and will report back here with some of my findings. Thanks for the reply!

**Comment by gwozdzm** [comment]
Hi Everyone, 

FerretDB maintainer here: as we're constantly improving our software, I'm wondering did you have a chance to test FerretDB + Payload? Any feedback is appreciated!

We recently released FerretDB version 1.5 with beta SQLite backend support:
https://blog.ferretdb.io/ferretdb-v-1-5-0-beta-support-sqlite/

**Comment by gwozdzm** [comment]
Hi Everyone,

FerretDB recently released 2.0 - much more performant and feature-rich software:
https://blog.ferretdb.io/ferretdb-releases-v2-faster-more-compatible-mongodb-alternative/
We should be close to achieving compatibility now - feel free to test it and share feedback!
